packages/node_modules/@webex/internal-plugin-conversation/src/share-activity.js

```diff
@@ -243,7 +310,7 @@ const ShareActivity = WebexPlugin.extend({
    * @returns {Promise<Object>}
    */
   prepare() {
-    if (!this.uploads.size) {
+    if (!(this.uploads.size + this.gifs.size)) {
```

Comment:
This would be more readable as !this.updloads.size && !this.gifs.size instead of doing the quick maths.

Reply:
Got it will do